src/app/modules/login/services/azure.ad.b2c.service.spec.ts

```diff
@@ -1,11 +1,12 @@
-import { TestBed, inject } from '@angular/core/testing';
+import { inject, TestBed } from '@angular/core/testing';
+import { UserAgentApplication } from 'msal';
+
 import { AzureAdB2CService } from './azure.ad.b2c.service';
-import { UserAgentApplication, Account } from 'msal';
 
 describe('AzureAdB2CService', () => {
   let service: AzureAdB2CService;
 
-  let account: Account;
+  let account;
```

Comment:
do we have a strong reason to remove the data type from here?

Reply:
I do not. Let me bring it back.